ffi/mupdf.lua

```diff
@@ -766,7 +766,7 @@ local function get_pthread()
     else
         -- Kobo devices strangely have no libpthread.so in LD_LIBRARY_PATH
         -- so we hardcode the libpthread.so.0 here just for Kobo.
-        for _, libname in ipairs({"libpthread.so", "libpthread.so.0"}) do
+        for _, libname in ipairs({"pthread", "libpthread.so.0"}) do
```

Comment:
I saw this before and wondered about changing it, but I figured there might've been a reason for it (though I can't think of any). :-)

Reply:
Yeah, benefit is that it can load `libpthread.dylib` on Mac now. Btw, this PR is developed solely on my Mac. Koreader is stable enough on Mac OSX now for development purpose. :)

Reply:
Cool. :)

Btw, I wanted to look into that Travis Mac build business, but at least on Travis that's not an option due to insufficient resources. See https://github.com/koreader/koreader-base/pull/564#issuecomment-339693544 (I imagine everything would fail spectacularly as written there, but that aside. I just wanted to play around with it a bit.)

Reply:
I'm not sure if the brew install stuff is sufficient. Probably there are hidden hacks on my Mac to make it work. Anyway I will verify that on a clean Mac when I have time.